### PR TITLE
docs(storybook): add versions selector in sidebar

### PR DESCRIPTION
**Pull request by feoche**

### CODE REVIEW

File: packages/tools/storybook/.storybook/manager.js [hunk 1]
```diff
@@ -2,6 +2,45 @@ import { addons } from '@storybook/addons';
...
+    }
+  }, 500);
+};
+
+setTimeout(() => {
+  const urlVersionRegex = /\/(latest|v(\d\.?){3}[^\/]*)\//gmi;
+  if (urlVersionRegex.test(location.href)) {
+    const releasesSelector = document.createElement('select');
+    releasesSelector.id = 'release-selector';
+    Object.entries(releases).map((value) => {
```
Comment (dpellier) [inline packages/tools/storybook/.storybook/manager.js:24]: prefer `forEach` if you don't care about what is returned

File: packages/tools/storybook/.storybook/manager.js [hunk 2]
```diff
@@ -2,6 +2,45 @@ import { addons } from '@storybook/addons';
...
+      const version = selection.target.value === 'latest' ? 'latest' : `v${selection.target.value}`;
+      location.pathname = location.pathname.replace(urlVersionRegex, version);
+    };
+  } else {
+    const version = document.createElement('div');
+    version.id = 'release-selector';
+    version.innerText = Object.keys(releases)[0];
+    createHeaderElement(version);
+  }
+}, 2000);
```
Comment (dpellier) [inline packages/tools/storybook/.storybook/manager.js:42]: why do you need to wait here?

File: packages/tools/storybook/.storybook/manager.js [hunk 3]
```diff
@@ -2,6 +2,45 @@ import { addons } from '@storybook/addons';
 import theme from './ods.theme';
 import React from 'react';
 
+import { releases } from '../public/releases.js';
+
+const createHeaderElement = (element) => {
+  const timer = setInterval(function(){
```
Comment (dpellier) [inline packages/tools/storybook/.storybook/manager.js:8]: 😧 is this really necessary?
can't you just add a dom loaded listener to start all your changes instead?

**Review by dpellier — CHANGES_REQUESTED** [review]
(careful with your branch name, it should relates to the work done)

I don't see the version select on the deployed doc, is it normal?

**Comment by feoche** [comment]
Outdated by #133

**Pull request closed without merging**